src/BTCPayServer.Lightning.CLightning/ShortChannelId.cs

```diff
@@ -0,0 +1,29 @@
+using System.Globalization;
+using System.Linq;
+
+namespace BTCPayServer.Lightning.CLightning
+{
+    // clightning-specific representation of channel id
+    public class ShortChannelId
+    {
+        public ShortChannelId (string data)
+        {
+            var numbers = data.Split(':').Select(s => int.Parse(s)).ToArray();
+            BlockHeight = numbers[0];
+            BlockIndex = numbers[1];
+            TxOutIndex = numbers[2];
+        }
+        public int BlockHeight { get; set; }
+        public int BlockIndex { get; set; }
+        public int TxOutIndex { get; set; }
+
+        public override string ToString()
+        {
+            return BlockHeight.ToString(CultureInfo.InvariantCulture) +
```

Comment:
use the syntax `return $"{BlockHeight.ToString(CultureInfo.InvariantCulture)}:{BlockIndex.ToString(CultureInfo.InvariantCulture)}:{TxOutIndex.ToString(CultureInfo.InvariantCulture)}";` it is easier to read, and more efficient.